client/src/components/Hamburger/Hamburger.stories.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+// import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import Hamburger from './Hamburger';
+
+// let typeName = "Bars";
+// let iconName = "bars"
+
+// function toggle(e) {
+//     e.preventDefault()
+//     if (iconName === "bars") {
+//         iconName = "times"
+//     } else {
+
+//     }
+// }
+
+// function toggle(e) {
+//     e.preventDefault();
+//     if (iconName === "bars"){
+//         iconName = "times"
+//         typeName = "Times"
+//         alert(iconName + typeName);
+//     }else{
+//         iconName = "bars"
+//         typeName = "Bars"
+//         alert(typeName);
+//     }
+    
+    
+// }
+
+
+storiesOf('Hamburger', module)
+    .add('Hamburger', () => 
+    <Hamburger type="icon" icon="bars">
+    </Hamburger>)
```

Comment:
I don't see where you're using these props. Do you need them?

Reply:
@ZakkMan , If i remove this code it won't render my button inside of storybook. Is there something else you want me to do, not sure how you want me to approach this